package-lock.json

```diff
@@ -0,0 +1,6374 @@
+{
+  "name": "agriculture-core",
```

Comment:
Since we are using yarn, this file is not necessary and shouldn't ever be generated. We could make a case to use npm for our dependencies instead of yarn, but we should only use one (or we have to run `yarn install && npm install` every time we make a pull request, effectively maintaining two lock files doing the same thing)